localstack-core/localstack/services/s3/utils.py

```diff
@@ -454,40 +442,6 @@ def base_64_content_md5_to_etag(content_md5: ContentMD5) -> str | None:
     return hex_digest
 
 
-def decode_aws_chunked_object(
```

Comment:
we did not use this code anymore, as `localstack.services.s3.codec.AwsChunkedDecoder` superseded it